app/views/layouts/application.html.erb

```diff
@@ -34,6 +34,20 @@
           </li>
         </ul>
         <div class="ms-auto d-flex">
+          <% if current_user.is_admin? %>
```

Comment:
This breaks the CSS for the `New Ticket` button a bit

Reply:
Just fixed